types/sentry__webpack-plugin/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for @sentry/webpack-plugin 1.9
+// Project: https://github.com/getsentry/sentry-webpack-plugin
+// Definitions by: Vladimir Grenaderov <https://github.com/VladimirGrenaderov>
+//                 Max Boguslavskiy <https://github.com/maxbogus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/seneca
+// TypeScript Version: 2.7
+
+import { Plugin } from 'webpack';
+
+declare class SentryPlugin extends Plugin {
+    constructor(options?: Options);
+}
+
+declare interface Options {
```

Comment:
nit: I'm guessing is not under your scope to add all of the options but I appreciate if you could.

Reply:
Good point. I added them.